adafruit_logging.py

```diff
@@ -41,54 +71,56 @@
     (50, "CRITICAL"),
 ]
 
-for value, name in LEVELS:
-    globals()[name] = value
+for __value, __name in LEVELS:
```

Comment:
Pycharm seems to think that these two lines of code define variables named "value" and "name" in the global scope. I'm not sure why. But adding the underscores in front makes sure that they're sorted at the bottom of code auto-completion results if so.

Reply:
Could you keep the variable names the same?